CHANGELOG

```diff
@@ -1,3 +1,6 @@
+* Mon Mar 25 2019 Joseph Sharkey <shark.bruhaha@gmail.com> - 0.2.0-0
```

Comment:
Why is this a minor version bump?  I would think this is a patch bump.

Reply:
It was new functionality for those fields so I wasn't sure what the cutoff for a patch bump was.

Reply:
It is not backwards compatible because the new datatype allows for `Array[String]` as well as `String`, and also allows for Integers, not just Strings.

Reply:
It is backward compatible (old settings still work with new type).